src/Compilers/Shared/BuildClient.cs

```diff
@@ -86,6 +89,37 @@ internal RunCompilationResult RunCompilation(IEnumerable<string> originalArgumen
             return new RunCompilationResult(exitCode);
         }
 
+        private static bool TryEnableMulticoreJitting(out string errorMessage)
+        {
+            errorMessage = null;
+            try
+            {
+                // Enable multi-core JITing
+                // https://blogs.msdn.microsoft.com/dotnet/2012/10/18/an-easy-solution-for-improving-app-launch-performance/
+                var profileRoot = Path.Combine(
+                    Environment.GetFolderPath(Environment.SpecialFolder.LocalApplicationData),
```

Comment:
>LocalApplicationData [](start = 72, length = 20)

Is this a per user folder? Is there an alternative that can be shared across all users?

Reply:
It is a per-user folder. Shared temporary storage isn't usually accessible for non-admin users. I could see this being a security risk, so I think we should be conservative for now.